week2/materials/homework/assignments.md

```diff
@@ -4,51 +4,61 @@
 
 Sync your changes to s3 and write down your s3 bucket url.
 
-bucket url: `?`
+bucket url: `http://keerthi-week2.s3-website-us-east-1.amazonaws.com`
 
 **Assignment 5:**
 
 Right now, the website does not support https. Explain which other AWS service needs to be integrated in order to achieve this.
 
-service name: `?`
+service name: `use CloudFront to serve HTTPS requests for my Amazon S3 bucket`
 
 **Assignment 6:**
 
 Figure out and write down the price per month of storing 51TB on S3.
 
-Price: `?`
+<!-- S3 Standard storage: 51 TB per month x 1024 GB in a TB = 52224 GB per month
+Pricing calculations
+Tiered price for: 52224 GB
+51200 GB x 0.0230000000 USD = 1177.60 USD
+1024 GB x 0.0220000000 USD = 22.53 USD
+Total tier cost: 1177.60 USD + 22.53 USD = 1200.1280 USD (S3 Standard storage cost)
+S3 Standard cost (monthly): 1,200.13 USD -->
+
+Price: `1,200.13 USD`
 
 Total size of website: `?`
 
 **Assignment 7:**
 
 Write down the main cost factors for S3
 
-brief description: `?`
+brief description: `There are six Amazon S3 cost components to consider when storing and managing your data - storage pricing, request and data retrieval pricing, data transfer and transfer acceleration pricing, data management and analytics pricing , and the price to process your data with S3 Object Lambda.`
 
 **Assignment 8:**
 
 The file `week2/assignments/products.json` needs to be uploaded Inside the bucket `hyf-serverless-course-week-2`. Upload the file through the CLI and write down the command needed.
 
-docs: 
+docs:`aws s3 cp D:\serverless\hyf-serverless-course\hyf-serverless-course\week2\materials\homework\products.json s3://keerthi-week2/`
+
+upload: ..\products.json to s3://keerthi-week2/products.json
 
 **Assignment 9:**
-Write down a brief use case on when S3 could be used for a data engineering assignment: 
+Write down a brief use case on when S3 could be used for a data engineering assignment:
 
 use case: `?`
 
 command:
 
-## Optional assignments: 
+## Optional assignments:
 
 **Assignment 10:**
 
 What can be done to reduce the pricing for S3 when hosting a large number of files?
 
-brief description: `?` 
+brief description: `Use binary format (e.g. AVRO) over human readable format (e.g. JSON), Using row-based vs. column based storage.`
```

Comment:
Well I was thinking the answer was more in the lines of this : https://aws.amazon.com/premiumsupport/knowledge-center/s3-reduce-costs/